backend/src/Edna/Setup.hs

```diff
@@ -10,14 +13,18 @@ module Edna.Setup
 
 import Universum
 
-import qualified Data.Char as C
-
-import Lens.Micro.Platform (makeLenses)
+import Data.ByteString as B
```

Comment:
This import brings the whole `Data.ByteString` into scope, it's prohibited by our style guide, i. e. all imports from external packages should be explicit. Perhaps you intended to make it `qualified`.

```suggestion
import qualified Data.ByteString as B
```

Reply:
yes, you are right! I wanted to add qualified import